### PR TITLE
Update test to remove decimal approximation in assertion

### LINKED ISSUE #14: Issue in test suite data
Great job on the implementation of carsons equations, it's a very useful reference.

I believe [this matrix](https://github.com/opusonesolutions/carsons/blob/6919cd5be416a58f14c1d5d933a52905a6d5f6a6/tests/test_carsons.py#L172-L188) in the tests is incorrectly ordered.

https://github.com/opusonesolutions/carsons/blob/6919cd5be416a58f14c1d5d933a52905a6d5f6a6/tests/test_carsons.py#L172-L188

If I use the carsons.py file to generate the `z_primitive_matrix` for `ABCN_geometry_line` I get the following:

```
0.00017479262640653616 + j 0.000859886093793189    5.9217626406536145e-05 + j 0.0004848090114987014    5.9217626406536145e-05 + j 0.0005291270819387303    5.9217626406536145e-05 + j 0.000488729331339374
5.9217626406536145e-05 + j 0.0004848090114987014    0.00017479262640653616 + j 0.000859886093793189    5.9217626406536145e-05 + j 0.0004514956068104968    5.9217626406536145e-05 + j 0.000476865015771444
5.9217626406536145e-05 + j 0.0005291270819387303    5.9217626406536145e-05 + j 0.0004514956068104968    0.00017479262640653616 + j 0.000859886093793189    5.9217626406536145e-05 + j 0.0004675586100839125   
5.9217626406536145e-05 + j 0.000488729331339374    5.9217626406536145e-05 + j 0.000476865015771444    5.9217626406536145e-05 + j 0.0004675586100839125    0.00042706962640653615 + j 0.0009609520589591509
```

If you make this change, you will be able to get rid of the `decimal=4` keyword argument in the assertion in your tests.

https://github.com/opusonesolutions/carsons/blob/6919cd5be416a58f14c1d5d933a52905a6d5f6a6/tests/test_carsons.py#L292-L295

### PR DESCRIPTION
**Pull request by kdheepak**
Resolves #14

### CODE REVIEW

**Comment by kdheepak** [comment]
I'm not sure why the tests don't pass on Travis-CI. They pass fine locally on my machine.

Screen Shot 2019-04-03 at 9 48 22 AM

**Comment by AnjoMan** [comment]
@kdheepak Thanks for the PR! This is a good catch. However, I think the issue is not the matrix order of the expected impedance results -- these are lifted from the [IEEE published results](http://sites.ieee.org/pes-testfeeders/files/2017/08/feeder13.zip). I think the issue is that the 601 line configuration has a different phase order. i was able to swap the phases [here](https://github.com/opusonesolutions/carsons/blob/6919cd5be416a58f14c1d5d933a52905a6d5f6a6/tests/test_carsons.py#L38) to 'ACBN' and the same result. Do you think you could rework your PR to modify the test inputs instead of the outputs? That way we can continue to test against a z_abc matrix that looks exactly like the published results.

**Comment by AnjoMan** [comment]
Also, the reason the Travis CI fails is that it runs `flake8` to check for linting issues. You can do
```
pip install flake8
flake8
```
...to reproduce this test locally.

**Comment by kdheepak** [comment]
Ah. That makes sense. Should I change the name of the class to `ACBN_geometry_line` as well?

**Comment by AnjoMan** [comment]
@kdheepak that would be a good idea! clearly its an easy thing to confuse.

**Comment by kdheepak** [comment]
flake8 results in a lot of changes, that were not introduced by me.

```
./carsons/carsons.py:56:31: E241 multiple spaces after ','
./carsons/carsons.py:56:54: E241 multiple spaces after ','
./carsons/carsons.py:133:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:166:13: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:173:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:174:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:175:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:176:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:177:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:178:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:179:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:180:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:181:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:182:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:183:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:184:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:185:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:186:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:187:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:188:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:193:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:194:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:195:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:196:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:197:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:198:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:199:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:200:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:201:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:202:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:203:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:204:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:205:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:206:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:207:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:208:24: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:215:13: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:223:13: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:230:13: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:230:17: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:230:26: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:230:35: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:230:44: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:231:17: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:231:26: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:231:35: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:231:44: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:232:17: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:232:26: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:232:42: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:232:64: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:233:17: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:233:26: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:233:41: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:233:65: E226 missing whitespace around arithmetic operator
./tests/test_carsons.py:251:17: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:258:13: E126 continuation line over-indented for hanging indent
./tests/test_carsons.py:268:17: E126 continuation line over-indented for hanging indent
./tests/test_multiple_neutrals.py:14:13: E241 multiple spaces after ':'
./tests/test_multiple_neutrals.py:15:13: E241 multiple spaces after ':'
./tests/test_multiple_neutrals.py:16:14: E241 multiple spaces after ':'
./tests/test_multiple_neutrals.py:25:53: E226 missing whitespace around arithmetic operator
./tests/test_multiple_neutrals.py:25:57: E226 missing whitespace around arithmetic operator
./tests/test_multiple_neutrals.py:26:53: E226 missing whitespace around arithmetic operator
./tests/test_multiple_neutrals.py:26:57: E226 missing whitespace around arithmetic operator
./tests/test_multiple_neutrals.py:42:13: E241 multiple spaces after ':'
./tests/test_multiple_neutrals.py:43:13: E241 multiple spaces after ':'
./tests/test_multiple_neutrals.py:44:13: E241 multiple spaces after ':'
```

I can fix these easily since I have a auto-formatter that does this. However, I see you have some recent branches and we may not want to make a lot of changes since then you may have to deal with some unnecessary merge conflicts.

**Comment by kdheepak** [comment]
Okay, that appears to have resolved the flake8 issues as well! This is ready for final review and to merge.

File: tests/test_dict_compatibility.py [hunk 1]
```diff
@@ -41,6 +41,6 @@ def __init__(self):
     z_primative = CarsonsEquations(model).build_z_primitive()
     assert_array_almost_equal(
         z_primative,
-        ABCN_line_z_primitive(),
+        ACBN_line_z_primitive(),
         decimal=4
```
Comment (AnjoMan) [inline tests/test_dict_compatibility.py]: Does your fix make this unnecessary? There are also two other places in the `test_carsons.py` file that have it, it seems like we could drop those as well now that we fixed this issue

Comment (kdheepak) [reply tests/test_dict_compatibility.py]: You are correct! I made the changes in 975c5cf.

**Pull request merged**